sections/main-product.liquid

```diff
@@ -359,6 +143,82 @@
                   {{ product.description }}
                 </div>
               {%- endif -%}
+              {%- when 'carousel' -%}
+                <cross-sell-carousel>
+                   <div class="glide" data-glide-cross-sell>
+                     <div class="glide__track" data-glide-el="track">
+                       <ul class="glide__slides">
+                         {% for product in collections[block.settings.collection].products %}
+                           <li class="glide__slide">
+                               <div class="card card--cross-sell">
+                               <img src="{{ product.featured_image | img_url: '134x134' }}"  alt="{{ product.title }}"/>
```

Comment:
Adding lazyloading to this image, and width and height attributes
an example of what width attribute would look like

`width="{{ product.featured_image.width }}"`